Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-settings.component.ts

```diff
@@ -52,7 +52,7 @@ export class TestSettingsComponent implements OnInit {
     isIpAddressAdded: boolean;
     isIpAddressFieldNull: boolean;
     
-    constructor(public dialog: MdDialog, private testService: TestService, private router: Router, private route: ActivatedRoute, private snackbarRef: MdSnackBar) {
+    constructor(public dialog: MdDialog, private testService: TestService, private router: Router, private route: ActivatedRoute, public snackbarRef: MdSnackBar) {
```

Comment:
why public?

Reply:
It has to be public in order to test whether the snackbar in the test-settings page has been opened or not. On being private, it cannot be accessed from the spec file.